solutions/gleam/01-init/code/src/redis.gleam

```diff
@@ -1,11 +1,16 @@
 import gleam/io
-import gleam/result
-import glisten/socket/options.{ActiveMode, Passive}
-import glisten/tcp
+
+import gleam/erlang/process
+import gleam/option.{None}
+import gleam/otp/actor
+import glisten
 
 pub fn main() {
-  use listener <- result.then(tcp.listen(6379, [ActiveMode(Passive)]))
-  use _socket <- result.then(tcp.accept(listener))
+  let assert Ok(_) =
```

Comment:
@lpil could you take a look at this - does this match the `glisten` usage you were suggesting?

Reply:
Looks great!